### DISCUSSION TITLE
Using zero-ui without proxy?

### DISCUSSION THREAD

**Discussion by seanbts** [dec0dOS/zero-ui#177] [Q&A]
Hi,

Is it possible to use zero-ui without the caddy proxy, we tunnel our traffic to cloudflare who provide the SSL cert.

Thanks, Sean

**Comment by aruznieto** [comment]
Yes, you can. To disable Caddy proxy and HTTPS, remove the `https-proxy` from `docker-compose.yml`, set `ZU_SECURE_HEADERS `to `false`, and change zero-ui port `expose` to `ports`.

After that you can create the CF tunnel.

**Comment by nityam2007** [comment]
try this one and change the stuff required 

removed caddy and disabled secure header  

version: "3"

services:
  zerotier:
    image: zyclonite/zerotier:1.10.6
    container_name: zu-controller
    restart: unless-stopped
    volumes:
      - ./zerotier-one:/var/lib/zerotier-one
    environment:
      - ZT_OVERRIDE_LOCAL_CONF=true
      - ZT_ALLOW_MANAGEMENT_FROM=0.0.0.0/0
    expose:
      - "9993"
    ports:
      - "9993:9993"
  zero-ui:
    image: dec0dos/zero-ui:latest
    container_name: zu-main
    build:
      context: .
      dockerfile: ./docker/zero-ui/Dockerfile
    restart: unless-stopped
    depends_on:
      - zerotier
    volumes:
      - ./zerotier-one:/var/lib/zerotier-one
      - ./data:/app/backend/data
    environment:
      - ZU_CONTROLLER_ENDPOINT=http://<PUBLICIP>:9993/
      - ZU_SECURE_HEADERS=false
      - ZU_DEFAULT_USERNAME=admin
      - ZU_DEFAULT_PASSWORD=admin
    expose:
      - "4000"
    ports:
      - "4000:4000"
